cli/lookup.ts

```
@@ -6,6 +7,7 @@ import {
   generateCryptoKeyPair,
   getAuthenticatedDocumentLoader,
   lookupObject,
+  type Object,
```

Biome (1.9.4)

[error] Do not shadow the global "Object" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)